### PR TITLE
mobile portfolio skeleton design

### PR DESCRIPTION
**Pull request by MishDM**
add portfolio images
add portfolio styles
add portfolio information

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+}
+
+.union {
+  margin-right: 24px;
+}
+
+.head-line {
+  background-image: url("/images/header-shapes.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
```
Comment (HAMMAS-SALEEM) [inline css/style.css:36]: ### Your Project on Production :bust_in_silhouette:
no-background-production (copy)


- [ ] Your background image is visible on the local server. :+1: But when I tried your application on the production site then there your background image was not visible. The problem is with your **URL**. Please fix the issue as I fixed it in the image below :arrow_down:

### Your Project on Local Machine
background-size-issue (copy)

- [ ] Your background is going out of resolution. Please fix the background-size: 100%; as suggested in the image below. :arrow_down_small:

### My fIx for both issues :point_down:
background-fix (copy)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    <div class="union" id="union">
+      <img src="images/union.svg" alt="union icon" />
+    </div>
+  </header>
+  <section class="head-line">
+    <h1 class="introduction first-text1">
+      I<span>&#8217;</span>m Misikir<br />
+      Glad to see <br />
+      you!
+    </h1>
```
Comment (HAMMAS-SALEEM) [inline index.html:27]: # Your Heading
Screenshot from 2022-07-02 03-17-41


# Figma Heading
heading-figma

- [ ] Please compare your heading with the Figma heading. Please make your heading as bold as in the Figma template. :+1:
- [ ] Please add `margin-bottom: 12px;` to your heading. :blush:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Changes Requested :recycle:

almost there

## Hey @MishDM :wave:
### Highlights :1st_place_medal:

Your project is working :clap:
Good commit history :heavy_check_mark:
Following Github Flow :heavy_check_mark:
Following HTML, CSS best practices :heavy_check_mark:
Following Figma template :art:
Using .gitignore file :heavy_check_mark:
Meaningful commit messages :heavy_check_mark:
Descriptive PR title :heavy_check_mark:
Descriptive PR Summary :+1:
Linters are green :green_circle:

You have made a great effort on your project :blush: and I can see that but you need to work on some issues to go to the next project but you are almost there. :+1:

### **_Any requested change with the optional tag is not crucial enough to prevent you from getting approval. But we, highly recommend that you to take these changes into consideration_**

### Once you implement all the changes you should be ready to submit your project for a re-review :rocket: :rocket:

Cheers and Happy coding!👏👏👏

### Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification. You can reach out to me on slack by searching my name (Hammas Bin Saleem) for any explanation.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mmsesay — COMMENTED** [review]
Hi @MishDM, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has falling linters.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by mmsesay — COMMENTED** [review]
Hi @MishDM, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has been merged without approval.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**